python/samples/04.ai.a.twentyQuestions/README.md

```diff
@@ -20,11 +20,7 @@ You can interact with the bot by asking questions to lead towards the secret wor
 
 ## Setting up the sample
 
-1. Clone the repository
```

Comment:
erring on the side of keeping all the existing instructions, and adding the quickstart link as an additional step (for both twentyQuestions and autogen - we can update all the other samples in another PR)